INSTALL.md

```diff
@@ -71,11 +72,49 @@ Aseprite can be compiled with two different back-ends:
 You will need the following dependencies:
 
     sudo apt-get update -qq
-    sudo apt-get install -y g++ libx11-dev libxcursor-dev cmake ninja-build
+    sudo apt-get install -y g++ libx11-dev libxcursor-dev cmake
+    
+Optionally, install ninja for parallelized builds: 
```

Comment:
We could put here something like "We use Ninja as the preferred build system, you can install it using:".